pkg/webui/console/store/middleware/logics/events.js

```diff
@@ -279,20 +272,10 @@ const createEventsConnectLogics = (reducerName, entityName, onEventsStart) => {
     }),
     createLogic({
       type: SET_EVENT_FILTER,
-      debounce: 250,
       process: async ({ action }, dispatch, done) => {
-        if (channel) {
-          try {
-            await channel.close()
-          } catch (error) {
-            if (isNetworkError(error) || isTimeoutError(action.payload)) {
-              dispatch(setStatusChecking())
-            } else {
-              throw error
-            }
-          } finally {
-            dispatch(startEvents(action.id, { silent: true }))
-          }
+        if (Boolean(channel)) {
+          await channel.close()
```

Comment:
`close` cannot throw anymore. It is also guaranteed now that by the time the `close` promise ends, the `close` event has been run (so the `dispatch` was done).